scripts/functions

```diff
@@ -1305,6 +1305,24 @@ CT_DoBuildTargetTuple() {
     CT_ARCH_TARGET_LDFLAGS="${CT_ARCH_TARGET_LDFLAGS} ${CT_ARCH_ENDIAN_LDFLAG}"
 }
 
+# This function determines the target tuple for a given set of compiler
+# flags, using either GCC's multiarch feature (if supported) or using
+# fallback architecture-specific functions.
+CT_DoMultilibTarget() {
+    local multi_flags="$1"
+    local target="$2"
+    local gcc_multiarch
+
+    gcc_multiarch=$("${CT_TARGET}-gcc" -print-multiarch ${multi_flags})
```

Comment:
> gcc won't choke - it just reports an empty string with 0 exit code

OK, so say so in the comment, maybe:

    # This function determines the target tuple for a given set of
    # compiler flags, using either GCC's multiarch feature if supported
    # (if not, gcc prints nothing and returns with 0), falling back to
    # calling the architecture-specific functions.